pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -4574,20 +4574,7 @@ func readRowsForTimeSeriesResult(rows driver.Rows, vars []interface{}, columnNam
 	var seriesList []*v3.Series
 	for _, key := range keys {
 		points := seriesToPoints[key]
-		// find the grouping sets point for the series
-		// this is the point with the zero timestamp
-		// if there is no such point, then the series is not grouped
-		// and we can skip this step
-		var groupingSetsPoint *v3.Point
-		for idx, point := range points {
-			if point.Timestamp <= 0 {
-				groupingSetsPoint = &point
-				// remove the grouping sets point from the list of points
-				points = append(points[:idx], points[idx+1:]...)
-				break
-			}
-		}
-		series := v3.Series{Labels: seriesToAttrs[key], Points: points, GroupingSetsPoint: groupingSetsPoint, LabelsArray: labelsArray[key]}
```

Comment:
The groping sets was primarily used for `Limit` clause in the metrics. However, we no longer rely on it and instead use the [ApplyMetricLimit](https://github.com/SigNoz/signoz/blob/592073a56426c757aa5c3660a376e4ce5453dff3/pkg/query-service/postprocess/limit.go#L13)